catroid/src/main/java/org/catrobat/catroid/drone/jumpingsumo/JumpingSumoDeviceController.java

```diff
@@ -20,41 +20,30 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.catrobat.catroid.content.actions;
+package org.catrobat.catroid.drone.jumpingsumo;
 
-import android.util.Log;
+import com.parrot.arsdk.arcontroller.ARDeviceController;
 
-import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
+public final class JumpingSumoDeviceController {
+	private static JumpingSumoDeviceController ourInstance = new JumpingSumoDeviceController();
```

Comment:
please reconsider the naming of this variable.. if you see it somewhere between the code you don't know at first what it is